tests/basic/pure_test.go

```diff
@@ -3505,6 +3505,386 @@ var _ = Describe("{IscsiPortsDownDuringNewPoolCreateInProgress}", func() {
+			if err != nil {
+				log.Errorf("Error creating pvc[%v], error[%v]", pvcName, err)
+			}
+			return err
+		}
+
+		sc := "phani"
+		ns := "testns"
+		pvc := "pvcwithtop"
+		err := createSC(sc)
+		log.FailOnError(err, fmt.Sprintf("Failed to create SC, error[%v]", err))
+		defer func() {
+			storage.Instance().DeleteStorageClass(sc)
+		}()
+		err = createNS(ns)
+		log.FailOnError(err, fmt.Sprintf("Failed to create NS, error[%v]", err))
+		err = createPVC(pvc, sc, ns)
+		log.FailOnError(err, fmt.Sprintf("Failed to create PVC, error[%v]", err))
+		defer func() {
+			core.Instance().DeletePersistentVolumeClaim(pvc, ns)
+			core.Instance().DeleteNamespace(ns)
+		}()
+
+		time.Sleep(10 * time.Second)
+		pvc1, err := core.Instance().GetPersistentVolumeClaim(pvc, ns)
+		log.InfoD("PVC pvc[%v] err[%v]", pvc1, err)
+		err = Inst().S.WaitForSinglePVCToBound(pvc, ns, 3)
+		log.FailOnError(err, fmt.Sprintf("error validating pvc"))
+	})
+	JustAfterEach(func() {
+		log.Infof("In Teardown")
+		defer EndTorpedoTest()
+	})
+
+})
+
+var _ = Describe("{FBDATopologyCreateTestNoRegion}", func() {
+	JustBeforeEach(func() {
+		log.Infof("Starting Torpedo tests ")
+		StartTorpedoTest("FBDATopologyCreateTestNoRegion",
+			"Try Creating FBDA pvcs using various topology options", nil, 0)
+	})
+	itLog := "FBDATopologyCreateTestNoRegion"
+	It(itLog, func() {
+		createSC := func(scName string) error {
+			params := make(map[string]string)
+			params["repl"] = "1"
+			params["priority_io"] = "high"
+			params["io_profile"] = "auto"
+			params["backend"] = "pure_file"
+			params["pure_nfs_endpoint"] = "10.9.126.67"
+			bindMode := storageApi.VolumeBindingImmediate
+			reclaimPolicyDelete := v1.PersistentVolumeReclaimDelete
+			v1obj := metav1.ObjectMeta{
+				Name: scName,
+			}
+
+			var zoneList []string
+			zoneList = append(zoneList, "zone-0")
+			/*
+			var regionList []string
+			regionList = append(zoneList, "region-0")
+			var req1 v1.TopologySelectorLabelRequirement
+			req1.Key = "toplogy.portworx.io/region"
+			list = append(list, req1)
+			req1.Values = regionList
+			*/
+			var list []v1.TopologySelectorLabelRequirement
+			var req v1.TopologySelectorLabelRequirement
+			req.Key = "toplogy.portworx.io/zone"
+			req.Values = zoneList
+			list = append(list, req)
+
+			var allowedTop []v1.TopologySelectorTerm
+			var top v1.TopologySelectorTerm
+			top.MatchLabelExpressions = list
+			allowedTop = append(allowedTop, top)
+			scObj := storageApi.StorageClass{
+				ObjectMeta:        v1obj,
+				Provisioner:       k8s.CsiProvisioner,
+				Parameters:        params,
+				ReclaimPolicy:     &reclaimPolicyDelete,
+				VolumeBindingMode: &bindMode,
+				AllowedTopologies: allowedTop,
+			}
+			storage.Instance().DeleteStorageClass(scName)
+			time.Sleep(1 * time.Second)
+			sc, err := storage.Instance().CreateStorageClass(&scObj)
+			log.Infof("Creation of sc[%v] got err[%v]", sc, err)
+			return err
+		}
+		createNS := func(nsName string) error {
```

Comment:
this function is also in common.go 

CreateNamespaces(nsName string, numberOfNamespaces int) ([]string, error)

Reply:
Done